src/components/questionaire.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const ZoneContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const QuizzContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  text-align: left;
+  margin-left: 3vw;
+  width: 40%;
+`;
+
+export default function Questionaire() {
+  const [YouAre, setYouAre] = useState();
+  const [YouWant, setYouWant] = useState();
+
+  const handleChange = (event) => {
+    setYouAre(event.target.value);
+  };
+
+  const handleChange2 = (event) => {
```

Comment:
change the name with a more specific name than handleChange2n exemple : handleChangeManager